sw/device/silicon_creator/rom_ext/rom_ext.c

```diff
@@ -406,6 +406,7 @@ static rom_error_t rom_ext_spx_verify(
     expected_root.data[i] ^= shares[i];
   }
 
+  dbg_printf("spx_verify: %C\r\n", key_alg);
```

Comment:
Is this debug print intended to be left in?

Reply:
No, I forgot to remove it.  Fixed.